src/keycloak_utils/consumer/rest_framework/core.py

```diff
@@ -0,0 +1,288 @@
+import socket
```

Comment:
import order not consistent, either use isort or handle manually. consider this comment for all imports across all files